src/Autofac.Extensions.DependencyInjection/AutofacRegistration.cs

```diff
@@ -63,11 +67,16 @@ public static class AutofacRegistration
         IEnumerable<ServiceDescriptor> descriptors,
         object? lifetimeScopeTagForSingletons)
     {
-        if (descriptors == null)
+        if (descriptors is null)
         {
             throw new ArgumentNullException(nameof(descriptors));
         }
 
+        if (builder is null)
+        {
+            throw new ArgumentNullException(nameof(builder));
```

Comment:
Codecov wants a unit test to cover the null `builder` case.

Reply:
Done. Added one for descriptors too.